### PR TITLE
Updated and Documented example.py and benchmarking.py

### PR DESCRIPTION
**Pull request by AmalNLal**
example.py 
1.  The issue with the generate query was resolved.
2. Documented the code

benchmarking.py
1.The initial parameters were fixed .There persists an issue with the query generator,for now a try and except block was added to make sure the code runs.
2.Documented the code.

### CODE REVIEW

File: .gitignore
Comment (Demirrr) [inline .gitignore:1]: Please do not include gitginore into repo

File: KGs/Animals/train.txt
Comment (Demirrr) [inline KGs/Animals/train.txt:1]: Please do not add datasets into repo

File: examples/multi_hop_query_answering/Experiments/2023-12-09 23-09-45.598470/configuration.json
Comment (Demirrr) [inline examples/multi_hop_query_answering/Experiments/2023-12-09 23-09-45.598470/configuration.json:1]: Please do not add experiments into repo

**Review by Demirrr — CHANGES_REQUESTED** [review]
Please do not add
1. datasets into repo
2. pretrained model configurations into repo

File: examples/multi_hop_query_answering/Experiments/2023-12-09 23-09-45.598470/configuration.json
Comment (AmalNLal) [reply examples/multi_hop_query_answering/Experiments/2023-12-09 23-09-45.598470/configuration.json:1]: I have created a new pull request

**Pull request closed without merging**